notes/health-care-evaluation.vbs

```diff
@@ -3997,360 +3952,606 @@ If HC_form_name = "No Form - Ex Parte Determination" Then
+		' Text 25, 45, 290, 20, "- For cases with a spenddown that do not meet an exception listed in EPM 2.3.4.2 MA-ABD Renewals, enter a date six months from the date updated in ELIG Renewal Date"
+		' Text 25, 65, 275, 10, "- For all other cases, enter the same date entered in the Elig Renewal Date"
+		' Text 10, 80, 145, 10, "- Exempt from 6 Mo IR: Enter N"
+		' Text 10, 95, 145, 10, "- ExParte: Enter Y"
+		' Text 10, 110, 255, 10, "- ExParte Renewal Month: Enter month and year of the ex parte renewal month"
+		EndDialog
+
+		'TO DO - update validation based on DHS direction for ex parte denial in phase 2
+		DO
+			Do
+				err_msg = ""    'This is the error message handling
+				Dialog Dialog1
+				cancel_confirmation
 
-	' MsgBox "About to CASE NOTE AND TIKL"
-	'If ex parte approved, create TIKL for 1st of processing month which is renewal month - 1
-	'TO DO - confirm TIKL information is correct
-	If ex_parte_determination = "Appears Ex Parte" Then Call create_TIKL("Phase 1 - The case has been evaluated for ex parte and appears to be ex parte on the information provided.", 0, DateAdd("M", -1, elig_renewal_date), False, TIKL_note_text)
+				' If ButtonPressed = hc_renewal_button Then Call check_hc_renewal_updates() ' TO DO - timing of function calls and completing function within loop?
 
-	If ex_parte_determination = "Case Transfered Out of County" Then script_end_procedure("Case List updated with Ex parte Evaluation. No CASE/NOTE as case is not in Hennepin County.")
-	'Navigate to and start a new CASE NOTE
-	Call start_a_blank_case_note
+				'TO DO - update with functions?
+				'Check the HC renewal screen data and compare against initial to ensure that changes made properly
 
-	'Add title to CASE NOTE
-	CALL write_variable_in_case_note("*** EX PARTE DETERMINATION - " & UCASE(ex_parte_determination) & " ***")
-
-	'For ex parte approval, write information to case note
-	If ex_parte_determination = "Appears Ex Parte" Then
-		CALL write_variable_in_case_note(TIKL_note_text)
-		CALL write_variable_in_case_note("Phase 1 - The case has been evaluated for ex parte and appears to be ex parte on the information provided. The case meets one of the criteria below.")
-		CALL write_variable_in_case_note("An MA-ABD enrollees will be ex parte renewed if their only source of income is:")
-		CALL write_bullet_and_variable_in_case_note("* ", "Supplemental Security Income (SSI), even if the benefit amount is zero")
-		CALL write_bullet_and_variable_in_case_note("* ", "Retirement, Survivors, and Disability Insurance (RSDI)")
-		CALL write_bullet_and_variable_in_case_note("* ", "SSI + RSDI")
-		CALL write_bullet_and_variable_in_case_note("* ", "Railroad Retirement Benefits (RRB)")
-		CALL write_bullet_and_variable_in_case_note("* ", "RSDI + RRB")
-		'TO DO - add additional language listing what would qualify for ex parte?
-		'TO DO - add additional case details - case number, renewal info, etc?
-	End If
+				'TODO - read DISA and display waiver
```

Comment:
There are some extras in here from the copy over - probably need to clean this up a bit.